features/step_definitions/authentication_steps.rb

```diff
@@ -93,10 +93,10 @@
 end
 
 Given /^I have a "([^\"]+)" cookie set to "([^\"]+)"$/ do |key, value|
-  headers = {}
-  Rack::Utils.set_cookie_header!(headers, key, value)
-  cookie_string = headers['Set-Cookie']
-  Capybara.current_session.driver.browser.set_cookie(cookie_string)
+    headers = {}
```

Comment:
Yes I added here the VCR.use_cassette and then remove it but forget to fix the white spaces. Sorry for that!